tests/Endpoints/SearchTest.php

```
@@ -501,17 +490,15 @@ public function testSearchSortWithString(): void
 
     public function testSearchSortWithInt(): void
     {
-        $response = $this->index->updateRankingRules([
+        $task = $this->index->updateRankingRules([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$task'. (undefined)

(UnusedLocalVariable)